docker-compose.yml

```diff
@@ -319,7 +315,7 @@ services:
 
 
 volumes:
-  data_lake:
+  # data_lake:  # Use bind-mount to inspect files, make sure data_lake/ exists at project root
```

Comment:
data_lake is used above but commented out here - is it initialized somwhere else?

Reply:
okay understood but where is the bind mount happening. Is the docker compose instruction changed?